packages/core/src/model-cartesian-charts.ts

```diff
@@ -11,6 +14,50 @@ export class ChartModelCartesian extends ChartModel {
 		super(services);
 	}
 
+	exportToCSV() {
+		const displayData = this.getDisplayData();
+		const options = this.getOptions();
+		const { groupMapsTo } = options.data;
+
+		const { cartesianScales } = this.services;
+		const domainIdentifier = cartesianScales.getDomainIdentifier();
+		const rangeIdentifier = cartesianScales.getRangeIdentifier();
+		const domainScaleType = cartesianScales.getDomainAxisScaleType();
+
+		let domainValueFormatter;
+		if (domainScaleType === ScaleTypes.TIME) {
+			domainValueFormatter = (d) => format(d, 'MMM d, yyyy');
+		}
+
+		// domain & range labels
+		const domainLabel = cartesianScales.getDomainLabel();
+		const rangeLabel = cartesianScales.getRangeLabel();
+
+		let data = [['Group', domainLabel, rangeLabel]];
+		data = data.concat(
+			displayData.map((datum) => [
+				datum[groupMapsTo], // group
+				datum[domainIdentifier] === null
+					? '–'
+					: domainValueFormatter
+					? domainValueFormatter(datum[domainIdentifier])
+					: datum[domainIdentifier], // domain value
+				datum[rangeIdentifier] === null
+					? '–'
+					: datum[rangeIdentifier].toLocaleString(), // range value
+			])
+		);
+
+		let csvString = '',
+			csvData = '';
+		data.forEach(function (d, i) {
+			csvData = d.join(';');
+			csvString += i < data.length ? csvData + '\n' : csvData;
+		});
```

Comment:
CSV files should be comma-separated, however from what I’ve read this can vary based on region. So semi-colon will not work in NA by default but may work in other regions? I think it might be best to use commas by default or hardcode the delimiter?

To hardcode the delimiter, change line 51 to:
```js
let csvString = 'sep=;\n',
```